content/accessibility/developer-toolkit.mdx

```diff
@@ -0,0 +1,77 @@
+
+A helpful learning and accountability tool is [Accessibility Insights](https://accessibilityinsights.io/). Any time you create a new or changed user-facing experience, run Accessibility Insights on the site or component to catch any accessibility errors.
+
+There are three different modes you can access to test in Accessibility Insights for Web, depending on the depth of testing needed: FastPass, Assessment, and AdHoc.
+
+### FastPass
+Within FastPass mode you have two tests to run.
+![Screenshot of automated checks feature in Accessibility Insights](https://user-images.githubusercontent.com/40274682/89364887-5b11a200-d688-11ea-85c2-016e9b326b28.png)
+- **Automated checks** scans the page for the most common accessibility errors, gives detailed location information, and helpful how to fix tips.
+	- You can file bugs/issues right from each of the accessibility errors.
+
+![Screenshot of the Tab Stops visual indicator on a page](https://user-images.githubusercontent.com/40274682/89364930-754b8000-d688-11ea-806d-5235f2836c4c.png)
+
+- **Tab stops** gives you a visual of the focus order on the page. It's not automated — you must tab through the page manually to get the visuals added.
+
+After running these tests you can export a report to share the status of the page.
+
+### Assessment
+![Screenshot of the Assessment feature on Accessibility Insights](https://user-images.githubusercontent.com/40274682/89364973-8a281380-d688-11ea-8430-1bc8e85b68db.png)
+Assessment is a manual but guided set of tests that covers all of WCAG 2.1 guidance. Because this can take some time, there is local storage for saving progress, and an overview page to help you keep track of progress. 
+
+You can print out an exported report of the assessment results at any time while testing.
+
+### AdHoc
+![Screenshot of AdHoc testing feature in Accessibility Insights](https://user-images.githubusercontent.com/40274682/89365128-d4a99000-d688-11ea-9db5-c29cb4a55075.png)
+AdHoc includes five overlay tests:
+
+- Automated checks (same test within each mode)
+- Landmarks
+- Color
+- Tab stops (same test within FastPass)
+- Headings
+
+![Screenshot of the helpful how to test information for the AdHoc tests on Accessibility Insights](https://user-images.githubusercontent.com/40274682/89365012-9f9d3d80-d688-11ea-8f26-bc918a20e34b.png)
+
+Each section has a link on understanding how to test, dos and don'ts, and best practices. 
+
+### Download:
+For the web/electron: 
+- [Accessibility Insights for Chrome](https://chrome.google.com/webstore/detail/accessibility-insights-fo/pbjjkligggfmakdaogkfomddhfmpjeni)
+- [Accessibility Insights for Edge](https://microsoftedge.microsoft.com/addons/detail/ghbhpcookfemncgoinjblecnilppimih)
+
+For mobile apps: 
+- [Accessibility Insights for Android](https://accessibilityinsights.io/en/downloads). 
+
+## Accessibility for teams
```

Comment:
Why is this title accessibility for teams?

Reply:
Ah. How about changing it to "Accessibility for EPD squads"? The link displays key things for each discipline to focus on.

Reply:
ah ok, the link is actually taking you straight to front-end dev, this one takes you to the same title you currently have: https://accessibility.digital.gov/

EPD is/can be internal jargon, so it's more like accessibility for different types of roles (although this would be a too long title) - can you think of something simpler? :)

Reply:
I put the link directly to the developer section since this is the developer toolkit. Should I change the title to "Accessibility for development teams"? And then I can put a link to the top level page for this resource somewhere else like general guidelines when I revamp it a bit more?

Reply:
Ah ok, sorry I'm trying to reply quickly so you're not waiting for me and didn't check which page we're on! 

I don't feel the page is "for teams" once you land in each of the different areas (design, front end, etc.). It's more of an accessibility guide, or checklist, for front end developers, that's why the title you had was confusing me.

Reply:
How about, "Accessibility guide for developers"?

Reply:
sounds good!